src/libs/actions/BankAccounts.js

```diff
@@ -363,54 +362,28 @@ function fetchFreePlanVerifiedBankAccount(stepToOpen) {
             const failedValidationAttemptsName = CONST.NVP.FAILED_BANK_ACCOUNT_VALIDATIONS_PREFIX + bankAccountID;
 
             // Now that we have the bank account. Lets grab the rest of the bank info we need
-            promiseAllSettled([
-                API.Get({
-                    returnValueList: 'nameValuePairs',
-                    name: failedValidationAttemptsName,
-                }),
-                API.Get({
-                    returnValueList: 'nameValuePairs',
-                    name: 'expensify_migration_2020_04_28_RunKycVerifications',
-                }),
-                API.Get({
-                    returnValueList: 'nameValuePairs',
-                    name: CONST.NVP.ACH_DATA_THROTTLED,
-                }),
-                API.Get({returnValueList: 'bankAccountList'}),
-                API.Get({
-                    returnValueList: 'nameValuePairs',
-                    name: CONST.NVP.BANK_ACCOUNT_GET_THROTTLED,
-                }),
-            ])
-                .then(([
-                    failedValidationAttemptsResponse,
-                    kycVerificationsMigrationResponse,
-                    achDataThrottledResponse,
-                    bankAccountListResponse,
-                    throttledBankAccountGetResponse,
-                ]) => {
+            API.Get({
+                returnValueList: 'nameValuePairs,bankAccountList',
```

Comment:
Can you add a space here? this just looks ugly 😅

Reply:
For you? Yea, I can do that.

Reply:
😘